frontend/src/components/HeaderApp.vue

```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useRouter } from 'vue-router';
+const router = useRouter();
+const userName = localStorage.getItem('username');
+const isLoggedIn = ref(userName !== null);
+const logout = () => {
+  localStorage.removeItem('username');
+  localStorage.removeItem('password');
+  localStorage.removeItem('email');
```

Comment:
Did you get the User API working? I can do this if not, but if you did please remove this localStorage stuff